pkg/config/cmd/cmd.go

```diff
@@ -57,47 +108,112 @@ func HaltOnError(fn AfterFunc) AfterFunc {
 	}
 }
 
-func encodeAndCreate(info *resource.Info, namespace string, obj runtime.Object) (runtime.Object, error) {
+// Create is the default create operation for a generic resource.
+func Create(info *resource.Info, namespace string, obj runtime.Object) (runtime.Object, error) {
 	return resource.NewHelper(info.Client, info.Mapping).Create(namespace, false, obj)
 }
 
-// Create attempts to create each item generically, gathering all errors in the
-// event a failure occurs. The contents of list will be updated to include the
-// version from the server.
-func (b *Bulk) Create(list *kapi.List, namespace string) []error {
-	resourceMapper := &resource.Mapper{ObjectTyper: b.Typer, RESTMapper: b.Mapper, ClientMapper: resource.ClientMapperFunc(b.RESTClientFactory)}
-	after := b.After
-	if after == nil {
-		after = func(*resource.Info, error) bool { return false }
+func NoOp(info *resource.Info, namespace string, obj runtime.Object) (runtime.Object, error) {
+	return info.Object, nil
+}
+
+func labelSuffix(set map[string]string) string {
+	if len(set) == 0 {
+		return ""
+	}
+	return fmt.Sprintf(" with label %s", labels.SelectorFromSet(set).String())
+}
+
+func CreateMessage(labels map[string]string) string {
+	return fmt.Sprintf("Creating resources%s", labelSuffix(labels))
```

Comment:
In new-app we always add at least one label - we print it so that
users have a hint to what they can group by.  We could potentially
omit it if user self enters.